src/components/sessions/list-sessions.astro

```diff
@@ -0,0 +1,108 @@
+---
+import ISO6391 from 'iso-639-1';
+
+const { sessions, lang, speakersData } = Astro.props;
+
+import { getLangFromUrl, useTranslations } from "../../i18n/utils";
+
+const currentLang = lang || getLangFromUrl(Astro.url);
+const t = useTranslations(currentLang);
+
+type Session = {
+  id: string;
+  title: string;
+  speakers: string[];
+  session_type: string;
+  track: string;
+  level: string;
+  language: string;
+  slug: string;
+};
+
+const speakerNames: Record<string, string> = speakersData 
+  ? Object.fromEntries(
+      Object.entries(speakersData).map(([id, speaker]: [string, any]) => [
+        id,
+        speaker.name
+      ])
+    )
+  : {};
+
+const getLanguageName = (code: string): string => {
+  const name = ISO6391.getName(code);
+  return name || code; // fallback to language code if not found
+};
+---
+
+<ol class="sessions space-y-6">
+  {
+    sessions.map(([key, session]: [string, Session]) => (
+      <li
+        class="session-item p-6 rounded-lg border-2 border-gray-200 dark:border-gray-700 hover:border-pinkpyladies dark:hover:border-pinkpyladies transition-all duration-200 bg-white dark:bg-gray-800 shadow-sm hover:shadow-md"
+        data-track={session.track || ""}
+        data-type={session.session_type || ""}
+        data-level={session.level || ""}
+        data-language={session.language || ""}
+      >
+        <div class="mb-4">
+          <h3 class="text-2xl font-bold mb-3">
+            <a 
+              href={`/${currentLang}/session/${session.slug}`} 
+              class="text-pinkpyladies hover:text-black dark:hover:text-white transition-colors duration-200 hover:underline decoration-2 underline-offset-4"
```

Comment:
```suggestion
              class="text-black dark:text-white hover:text-black dark:hover:text-white transition-colors duration-200 hover:underline decoration-2 underline-offset-4"
```
There is too much red in the current styling, do you mind if we just use black/white ?